packages/components/src/Tile/src/TileGroup.tsx

```diff
@@ -0,0 +1,122 @@
+import { useResponsiveValue, useStyledSystem, type ResponsiveProp, type StyledComponentProps } from "@hopper-ui/styled-system";
+import clsx from "clsx";
+import { Children, forwardRef, type CSSProperties, type ForwardedRef } from "react";
+import { Provider, ToggleButtonGroup, useContextProps, type Key } from "react-aria-components";
+
+import { cssModule, type BaseComponentDOMProps } from "../../utils/index.ts";
+
+import { InternalTileContext } from "./TileContext.ts";
+import { TileGroupContext } from "./TileGroupContext.ts";
+
+import styles from "./TileGroup.module.css";
+
+export const GlobalTileGroupCssSelector = "hop-TileGroup";
+
+export interface TileGroupProps extends
+    StyledComponentProps<BaseComponentDOMProps> {
+    /**
```

Comment:
I'd simplify this and allow any ToggleButtonGroup props. no reason to force single selection, it's "free" to allow multiple selection, so why not. We can set the single selection as the default

Reply:
We should set the default to single slection + disallowEmptySelection to true to fit the design guidelines